app/routes/event.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express')
+const auth = require('../middleware/auth')
+const router = express.Router()
+const eventController = require('../controllers/event')
+
+// CREATE A POST
+router.post(
+  '/',
+  auth,
+  eventController.createEvent
+)
+
+router.post(
+  '/event_update/:id',
+  auth,
+  eventController.updateEvent
```

Comment:
the update request should be a `PATCH` request.
Read More here:
https://stackoverflow.com/questions/31089221/what-is-the-difference-between-put-post-and-patch
https://rapidapi.com/blog/put-vs-patch/